core/src/main/java/de/danoeh/antennapod/core/storage/ApDownloadAlgorithm.java

```diff
@@ -16,10 +16,10 @@
 
 /**
  * Implements the automatic download algorithm used by AntennaPod. This class assumes that
- * the client uses the APEpisodeCleanupAlgorithm.
+ * the client uses the {@link EpisodeCleanupAlgorithm}.
  */
-public class APDownloadAlgorithm implements AutomaticDownloadAlgorithm {
-    private static final String TAG = "APDownloadAlgorithm";
+public class ApDownloadAlgorithm {
```

Comment:
I think it would be more clear to rename to `AutomaticDownloadAlgorithm`

Reply:
Good idea, I renamed the method.